events/tests/test_api.py

```diff
@@ -2290,3 +2290,114 @@ def test_event_ticket_system_password_not_own_child(guardian_api_client):
         another_child.ticket_system_passwords.get(event=event)
         == another_childs_password
     )
+
+
+VERIFY_TICKET_QUERY = """
+  query VerifyTicket($referenceId: String!){
+    verifyTicket(referenceId:$referenceId){
+      enrolment {
+        occurrence {
+          time
+        }
+        child {
+          birthdate
+        }
+      }
+      validity
+    }
+  }
+"""
+
+
+# NOTE: api_client is for anonymous users
+def test_verify_valid_ticket(api_client, snapshot):
```

Comment:
I guess this query should not have access to `child` if it's unauthenticated

Reply:
you are absolutely right! Could not figure out any other good solution than replace the whole enrolment object with 2 new fields: event name and occurrence time